Corona-Warn-App/src/main/res/layout/include_submission_done_no_consent.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:bind="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:focusable="true">
+
+            <ImageView
+                android:id="@+id/submission_done_hero_illustration"
+                android:layout_width="@dimen/match_constraint"
+                android:layout_height="wrap_content"
+                android:focusable="true"
+                android:src="@drawable/ic_illustration_together"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                bind:cwaContentDescription="@{@string/submission_done_no_consent_illustration_description}"
+                tools:ignore="ContentDescription" />
```

Comment:
What is the bind needed for here?

Reply:
I believe this is also for Talkback accessbility.

Reply:
The bind combines a content description with it's type, e.g. "No Consent" + "Button". I'm not a fan of this, but have not had the chance to look into whether this can be changed, I just replaced the previous solution with the binding to remove static context access.